src/main.ts

```diff
@@ -75,24 +76,33 @@ const initLoad = async () => {
         .then(async (file) => {
             window.console.debug('Loaded config.json')
 
-            await store.dispatch('importConfigJson', file)
+            store.dispatch('importConfigJson', file)
             if ('defaultLocale' in file) i18n.locale = file.defaultLocale
+
+            // Handle theme outside of store init and before vue mount for consistency in dialog
+            let theme = file.defaultTheme ?? defaultTheme
+            const uiSettingsUrl =
+                store.getters['socket/getUrl'] + '/server/database/item?namespace=mainsail&key=uiSettings'
```

Comment:
This will not work, because in some modes, you don't have a socket URL right now. you can only store a default theme setting in the config.json itself.

for example in my.mainsail.xyz. here will be only load the setting "instanceDB: browser" here without any moonraker connection. only after add/select a printer in the printer select dialog, there exists a moonraker url.